File: API/Features/PickupPoints/Dtos/PickupPointReadDto.cs
using API.Features.CoachRoutes;

namespace API.Features.PickupPoints {

    public class PickupPointReadDto {

        public int Id { get; set; }
        public string Description { get; set; }
        public string ExactPoint { get; set; }
        public string Time { get; set; }
        public string Remarks { get; set; }
        public bool IsActive { get; set; }
        public string User { get; set; }
        public string LastUpdate { get; set; }

        public CoachRouteAutoCompleteVM CoachRoute { get; set; }

    }

}

File: FrontEnd/src/app/features/pickupPoints/classes/dtos/pickupPoint-read-dto.ts
import { CoachRouteAutoCompleteVM } from 'src/app/features/coachRoutes/classes/view-models/coachRoute-autocomplete-vm'
import { Metadata } from 'src/app/shared/classes/metadata'

export interface PickupPointReadDto extends Metadata {

    // PK
    id: number
    // Object fields
    coachRoute: CoachRouteAutoCompleteVM
    // Fields
    description: string
    exactPoint: string
    time: string
    remarks: string
    isActive: boolean

}
